src/web/views/check.njk

```diff
@@ -70,10 +70,10 @@
           },
           {
             text: [
-              claim.cardDeliveryAddress.addressLine1,
-              claim.cardDeliveryAddress.addressLine2,
-              claim.cardDeliveryAddress.townOrCity,
-              claim.cardDeliveryAddress.postcode
```

Comment:
I think I preferred having claim.cardDeliveryAddress stored in the the claim, but this ties in nicely with the changes I've been making. If we get this merged in then I'll update my pr to store details in claim.cardDeliveryAddress without updating the request body